modules/cart/src/CartManager.php

```diff
@@ -118,6 +129,13 @@ public function addLineItem(OrderInterface $cart, LineItemInterface $line_item,
       $needs_cart_save = TRUE;
     }
 
+    // If the language is different, we need to save it again. If we are already
+    // planning to trigger the save, there is no need to do the additional
+    // checks.
+    if (!$needs_cart_save && $cart->getLanguage()->getId() <> $this->languageManager->getCurrentLanguage()->getId()) {
```

Comment:
I don't see any usage of `<>` in the php code, I think this should be replaced by `!==`